ushahidi/platform-client
# fixing totals-bug

NOTE: This should be merged to release/cycle-3-sms-csv as well.
This pull request makes the following changes:
- Removes old cost-code that was stopping the total number of messages to be calculated
- Removes a hyphen from notification-window
Testing checklist:
- Create a new targeted survey
- [ ] In step 4, the message should read We will send <number of questions> questions to <number of people> people via SMS for a total <number of questions * number of people> of messages
- click on Publish survey
- [ ] In the confirm-modal the same text should appear
- [ ] In the notification-window, no '-' should be present in "telecommunications provider"


Fixes ushahidi/platform#2530 .

Ping @ushahidi/platform
